src/components/override-sdk/infra/Assignment/Assignment.tsx

```diff
@@ -98,6 +98,21 @@ export default function Assignment(props) {
     setServiceShutteredStatus(serviceShuttered);
   }, [serviceShuttered]);
 
+  // Sets the language for the texts and emails if the user changes the language before opening an existing claim.
+  function initialLanguageCall() {
```

Comment:
Can you try moving this logic in claimlist tsx insetad of Assignment.tsx, as it is common for all?

Reply:
I did try to do that. But as the claim isn't actually open when you are in the Claimslist you aren't allowed to update the language, it's a 'Bad request' at that point when you re-open the claim. So it has to happen when the claim is actually open unfortunately.